examples/tensorflow_synthetic_benchmark.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
What's performance of this benchmark compared to native TF benchmark?

Reply:
On 16 GPUs (8 nodes), I got 146 im/s with PyTorch, 140 im/s with native TF benchmark, and 135 im/s with this Horovod benchmark.

Reply:
How about single node?

Reply:
189 im/s with the Horovod benchmark vs 195 im/s with the TF native benchmark.

Reply:
Sweet! I suggest to pull out those #s in the PR description.  They're really nice - TF native benchmark is way more bloated.